rcsb/utils/dictionary/DictMethodEntryHelper.py

```diff
@@ -762,27 +774,28 @@ def addEntryInfo(self, dataContainer, catName, **kwargs):
                 xObj = dataContainer.getObj("struct")
                 if xObj.hasAttribute("pdbx_structure_determination_methodology"):
                     methodType = xObj.getValue("pdbx_structure_determination_methodology", 0)
+            if not methodType:
+                if dataContainer.exists("exptl"):
+                    methodType = "experimental"
+                if dataContainer.exists("ma_data"):
+                    methodType = "computational"
             #
             if dataContainer.exists("exptl"):
                 xObj = dataContainer.getObj("exptl")
                 entryId = xObj.getValue("entry_id", 0)
                 methodL = xObj.getAttributeValueList("method")
                 methodCount, expMethod = self.__commonU.filterExperimentalMethod(methodL)
-                if not methodType:
-                    methodType = self.__commonU.filterStructureDeterminationMethodType(methodL)
```

Comment:
Was there a strong reason to not use this method instead? This method was intended to also determine if it's integrative...

Reply:
Thought it is better to use a top-level category to determine if the entry is `experimental` or `computational`. Also, what if `_ma_model_list.model_type` is a `.` in the ModelCIF file?